src/models/UserModel.js

```diff
@@ -0,0 +1,15 @@
+// api/v1/models/userModel.js
+import { pool } from "../config/database.js";
+
+export const createUser = async (firstname, lastname, email, password) => {
+	const query =
+		"INSERT INTO users (firstname, lastname, email, password) VALUES ($1, $2, $3, $4) RETURNING *";
+	const values = [firstname, lastname, email, password];
```

Comment:
The password should not be saved as plain text. It should be hashed. Use the [bcrypt](https://www.npmjs.com/package/bcrypt) package